### DISCUSSION TITLE
The return button in the built-in osu!lazer browser

### DISCUSSION THREAD

**Discussion by durshlakk** [ppy/osu#25002] [Ideas] [answered]
Greetings, dear developers! I want to offer you the idea of a button to return to the previous page of the built-in browser osu!lazer. When, for example, you go to the mapset of interest in the map library and view information about it, I would like to go back to the map library tab without unnecessary movements with one return button, otherwise right now you need to close the fully built-in browser and go back to the map library. This was an example, and so this button is needed everywhere, even, for example, when you go to someone's profile and would like to return to the tab back where you entered it. I hope you can hear me. Thank you in advance!

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/5063.
